linera-client/src/client_options.rs

```diff
@@ -44,6 +44,10 @@ pub struct ClientContextOptions {
     #[arg(long = "wallet")]
     pub wallet_state_path: Option<PathBuf>,
 
+    /// Sets the file storing the keystore state.
+    #[arg(long = "keystore")]
```

Comment:
As a small but significant naming change, I think that after this split what you've called ‘keystore’ is what other people like to call ‘wallet’, and what we're now calling ‘wallet’ is actually some kind of client state+configuration.

Reply:
To be honest, I don't think there's a fully-agreed upon dictionary here. Some people call the whole "blockchain" as _ledger_/wallet. I think we can fix the names later.